pkg/apis/hco/v1beta1/hyperconverged_types.go

```diff
@@ -46,10 +46,36 @@ type HyperConvergedSpec struct {
 	// +optional
 	LiveMigrationConfig LiveMigrationConfigurations `json:"liveMigrationConfig,omitempty"`
 
+	// certConfig holds the rotation policy for internal, self-signed certificates
+	// +optional
+	CertConfig *HyperConvergedCertConfig `json:"certConfig,omitempty"`
+
 	// operator version
 	Version string `json:"version,omitempty"`
 }
 
+// CertConfig contains the tunables for TLS certificates
```

Comment:
I think we should also mention that all the durations should be expressed in a string format that comply with golang parseDuration string format ( https://golang.org/pkg/time/#ParseDuration ).

Reply:
done